cfgov/templates/welcome.html

```diff
@@ -3,14 +3,42 @@
 
 {% block furniture %}
     <div class="content-wrapper">
+
         <h1>{% trans 'Welcome' %} <br/>
-        You've logged in to the consumerfinance.gov editorial tools.	
+        {% if 'build' in request.site.hostname %}
```

Comment:
Not sure I understand correctly. I am actually purposefully asking that the build version of Wagtail have different welcome language to reinforce that it's not actually connected to the live site. Just like the change to the log in screen that says "Log in to demo Wagtail" instead of "Log in to consumerfinance.gov".